sunxspex/emission.py

```diff
@@ -115,12 +116,12 @@ def flux(self, electron_energy):
         index = np.where((electron_energy < self.eebrk) & (electron_energy >= self.eelow))
         if index[0].size > 0:
             res[index] = self._norm_factor * self._n0 * (self.p - 1.) \
```

Comment:
Instead of using the line break, we should use brackets to split over lines.  That should make the multi-line alignment nicer too.

Reply:
I'm not sure what you mean throw it into brackets `( ... )`? I think this was done by autopep8